<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShareScreenWidget</class>
 <widget class="QWidget" name="ShareScreenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: 'Microsoft YaHei';
	font-size:16px;
}
#ShareScreenWidget{
	background:#1d2129;
}

#lbl_info{
	font-family: 'Microsoft YaHei';
	font-size:16px;
	color:#fff;
}
#btn_close{
	border:none;
	color:#fff;
}

QScrollArea
{
border:none;
}
#scrollAreaWidgetContents{
background:#1d2129;
}

QScrollBar:vertical 
{
border: none;
background-color:rgb(0,0,0);
width: 4px;margin: 0px 0 0px 0;
}
QScrollBar::handle:vertical { 
	background:#4E5969; 
    min-height: 20px;
	width: 4px;
	border-radius:2px;
}
       QScrollBar::add-line:vertical {background-color: 
      rgb(0,0,0);height: 4px;}
      QScrollBar::sub-line:vertical {background-color: 
      rgb(0,0,0);height: 4px;}
  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical 
      {height: 0px;}
       QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
      {background: none;}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>32</number>
     </property>
     <property name="topMargin">
      <number>24</number>
     </property>
     <property name="rightMargin">
      <number>32</number>
     </property>
     <property name="bottomMargin">
      <number>24</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_info">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>选择共享内容</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>432</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>24</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ShareViewContainer" name="screen_views" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ShareViewContainer" name="window_views" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShareViewContainer</class>
   <extends>QWidget</extends>
   <header>share_view_container.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
